File: cpp_e9zMZ/cpp/05-uocso-boiso/104.cpp
#include <algorithm>
#include <iostream>
using namespace std;

int main() {
    int x = 25;
    int y = 10;
    int result = 0;
    int minNumber = min(x, y);
    
    for (int i = minNumber; i >= 1; i--) {
        if (x % i == 0 && y % i == 0) {
            result = i;
            break;
        }
    }

    cout << "UCLN của " << x << " và " << y << " là " << result;
    return 0;
}

File: python_ToSXs/python/05-uocsoboiso/104.py
x = 27
y = 18

result = 0
zmin = min(x, y)

for i in range(zmin, 0, -1):
    if x % i == 0 and y % i == 0:
        result = i
        break

print(f"UCLN của {x} và {y} là {result}")
